webapp/src/components/post_type_jitsi/post_type_jitsi.tsx

```diff
@@ -103,7 +102,7 @@ export class PostTypeJitsi extends React.PureComponent<Props, State> {
         }
 
         meetingLink += `#config.callDisplayName="${props.meeting_topic || props.default_meeting_topic}"`;
-        meetingLink += `&userInfo.displayName="${getFullName(this.props.currentUser)}"`;
+        meetingLink += `&userInfo.displayName="${this.props.currentUser.username}"`;
```

Comment:
Why we are using always the username instead of the full name? Also, shouldn't this be url encoded?

Reply:
@larkox This was done in favour of [issue](https://github.com/mattermost/mattermost-plugin-jitsi/pull/240), in which we have to use the username of the user. Let me know if I need to update this to full name

Reply:
I agree about the url encoded comment